packages/server/graphql/public/mutations/helpers/movePageToNewParent.ts

```diff
@@ -14,6 +15,10 @@ export const movePageToNewParent = async (
   parentPageId: number
 ) => {
   const pg = getKysely()
+  const viewerAccess = await pageAccessByUserId({} as any).load({pageId, userId: viewerId})
```

Comment:
-1 let's not do this. If we can't use a dataloader here, let's move the page access by user id query to a common place and just call it.
Also, if we can't use a dataloader here, this function probably shouldn't be in `mutation/helpers`.